dask_planner/src/sql/table.rs

```diff
@@ -25,12 +25,20 @@ use crate::{
 /// DaskTable wrapper that is compatible with DataFusion logical query plans
 pub struct DaskTableSource {
     schema: SchemaRef,
+    #[allow(dead_code)]
```

Comment:
Curious why this is needed

Reply:
Rust currently throws up warnings that the `statistics` field isn't being used anywhere else (which shows up in our CI), but with the Join Reordering PR the warning will go away.